jpype/_pyinstaller/hook-jpype.py

```diff
@@ -0,0 +1,13 @@
+import os
+from pathlib import Path
+
+from PyInstaller.utils.hooks import relpath_to_config_or_make
+
+import jpype
+
+
+fspath = getattr(os, 'fspath', str)
+
+jar_path = Path(jpype.__file__).parent.parent.joinpath('org.jpype.jar')
```

Comment:
We will need additional jars in the future as jpype is likely to depend on asm for things like object extension.   But this is sufficient for now.  We can add a glob based pattern when the number of items increases.